cfme/tests/control/test_basic.py

```diff
@@ -55,14 +55,36 @@ def random_alert():
     alert.delete()
 
 
-@pytest.yield_fixture(params=[explorer.VMCompliancePolicy,
-                              explorer.HostCompliancePolicy,
-                              explorer.HostControlPolicy,
-                              explorer.VMControlPolicy],
-                      ids=["VMCompliancePolicy",
-                           "HostCompliancePolicy",
-                           "HostControlPolicy",
-                           "VMControlPolicy"])
+@pytest.yield_fixture(
+    params=[
+        explorer.HostCompliancePolicy,
+        explorer.VMCompliancePolicy,
+        explorer.ReplicatorCompliancePolicy,
+        explorer.PodCompliancePolicy,
+        explorer.ContainerNodeCompliancePolicy,
+        explorer.ContainerImageCompliancePolicy,
+        explorer.HostControlPolicy,
+        explorer.VMControlPolicy,
+        explorer.ReplicatorControlPolicy,
+        explorer.PodControlPolicy,
+        explorer.ContainerNodeControlPolicy,
+        explorer.ContainerImageControlPolicy
+    ],
+    ids=[
```

Comment:
Now it got quite long and I do believe the ids= can also take callables, so something like:
```python
ids=lambda policy_class: policy_class.__name__
```

should do this typing for you